templates/macros/helpers.html

```diff
@@ -27,3 +27,15 @@
   } %}
 <a href="mailto:{{email}}{% if subject %}?subject={{ subject|urlencode }}{% endif %}" {{attr|xmlattr}}>{{email}}</a>
 {%- endmacro -%}
+
+{%- macro interviewer_note(interviewer_only='False', title='', type='') -%}
+  {% if (interviewer_only and (type == 'Interstitial')) %}
+    <h1 class="question__title"><mark class="instruction">{{_("Interviewer Note:")}}</mark> {{ title }}</h1>
+  {% elif (title and (type == 'Interstitial')) %}
```

Comment:
Again, the brackets shouldn't be needed.

Reply:
Removed👍